sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -66,6 +66,43 @@ export class ChangeFeedIterator<T> {
     get hasMoreResults(): boolean;
 }
 
+// @public
+export interface ChangeFeedIteratorOptions {
+    // Warning: (ae-forgotten-export) The symbol "ChangeFeedResource" needs to be exported by the entry point index.d.ts
+    changeFeedResource?: ChangeFeedResource;
```

Comment:
The problem with separating the Resource and the StartType is that a user can mix them up incorrectly, for example, use a Continuation that was generated from ResourceType Full Container and that might generate.

On Java and .NET we went with the public API that is always StartFrom and the options are:

* Start from continuation
* Start from beginning - All container / EPK (FeedRange)
* Start from time - All container / EPK (FeedRange)
* Start from now - All container / EPK (FeedRange)

That way its impossible to mix a continuation with a mode that it's not related to it

Reply:
Thanks for the feedback. I have changed it now and made it similar to .NET/Java SDK. 
Eg. `ChangeFeedStartFrom.Beginning(partitionKey)`